hr_holidays_compute_days/README.rst

```diff
@@ -6,30 +6,49 @@
 Employee Compute Leave Days
 ===========================
 
-Computes the actual days for which employee will be on leave taking into 
-account both rest days and public holidays
+Computes the actual days for which employee will be on leave taking into
+account:
 
-Installation
-============
-
-No additional steps necessary
+* Other leaves in the same interval.
```

Comment:
Remove the dot at the end of this line.

Reply:
Why? :thinking:

Reply:
I think it should be consistent between the 3 bullet points.

Reply:
In that case it's the period in the next line what's missing :slightly_smiling_face: